contrib/examples/prometheus/prometheus.yml

```diff
@@ -0,0 +1,341 @@
+#
+# Prometheus will run uner the "prometheus" Service Account, create
```

Comment:
typo: under

Reply:
Thanks for the review @pmorie, typos matter.  All have bee addressed.